app/src/renderer/components/wallet/LiCoin.vue

```diff
@@ -12,9 +12,17 @@
           {{ amount }}
         </p>
       </div>
+      <!-- disable send on the hub until send is enabled -->
+      <tm-btn
+        v-if="!lastHeader || lastHeader.chain_id === 'cosmoshub-1'"
```

Comment:
shouldn't this be check if `production` instead of checking the `chain_id`?

Reply:
no production is also the testnet